JavaScriptFileType.js

```diff
@@ -172,6 +172,13 @@ JavaScriptFileType.prototype.write = function(translations, locales) {
                     db.getResourceByCleanHashKey(res.cleanHashKeyForTranslation(langDefaultLocale), function(err, translated) {
                         if (translated) {
                             baseTranslation = translated.getTarget();
+                        } else if (!translated && this.isloadCommonData) {
```

Comment:
Line 173 is if translated, so you don't have to put if not translated in the else condition.